pkg/environment/config.go

```diff
@@ -30,15 +30,23 @@ type Config struct {
 	RolloutUpgrade struct {
 		TestFarms []uint32 `json:"test_farms"`
 	} `json:"rollout_upgrade"`
-	RelaysURLs    []string `json:"relays_urls"`
 	SubstrateURL  []string `json:"substrate_urls"`
+	RelaysURLs    []string `json:"relays_urls"`
 	ActivationURL []string `json:"activation_urls"`
 	GraphQL       []string `json:"graphql_urls"`
 	KycURL        string   `json:"kyc_url"`
 	RegistrarURL  string   `json:"registrar_url"`
-	// FlistURL      []string `json:"flist_url"`
-	// HubURL        []string `json:"hub_url"`
-	BinRepo string `json:"bin_repo"`
+	BinRepo       string   `json:"bin_repo"`
+	GeoipURLs     []string `json:"geoip_urls"`
+
+	HubURL   []string `json:"hub_url"`
+	V4HubURL []string `json:"v4_hub_url"`
+
+	// we should not be supporting flist url or hub storage from zos-config until we can update them on runtime
+	// FlistURL   string `json:"flist_url"`
```

Comment:
let's have this flisturl configurable just in case we lost our urls again we can boot new machines